Missing `toml` package in `rollup-plugin-rust`

**Describe the bug**
The `yarn build` command execution returns the following message:

```bash
➤ YN0000: (!) Plugin typescript: @rollup/plugin-typescript TS2307: Cannot find module 'toml' or its corresponding type declarations.
➤ YN0000: src/index.ts: (4:18)
➤ YN0000: 
➤ YN0000: 4 import toml from "toml"
➤ YN0000:                    ~~~~~~
```

--- a/packages/rollup-plugin-rust/package.json
+++ b/packages/rollup-plugin-rust/package.json
@@ -37,6 +37,7 @@
         "glob": "^7.1.6",
         "rimraf": "^3.0.0",
         "rollup-pluginutils": "^2.8.2",
+        "toml": "^3.0.0",
         "wasm-pack": "^0.10.2"
     }
 }

